pkg/web/apps/apps_http_handler_test.go

```diff
@@ -378,3 +378,53 @@ func Test_httpHandler_GetActiveAppByID(t *testing.T) {
 		})
 	}
 }
+
+func TestHTTPHandler_InitClientApp(t *testing.T) {
+	deviceWithMissingDeviceID := helpers.GetMockDevice() // See that just one mock is enough
+	deviceWithMissingDeviceID.DeviceID = ""              // See here that you can change the object to match with the criteria
+
+	deviceWithMissingVersion := helpers.GetMockDevice()
```

Comment:
This is local variables. You can create many local variables for specific scenarios of the test.